cddl/linear-privilege-level.cddl

```diff
@@ -0,0 +1,8 @@
+linear-privlevel-type-choice /= signed-integer-range-type-choice
+signed-integer-range-type-choice = int-eq / int-range / tagged-int-eq / tagged-int-range
+int-eq = int
+int-range = [min: inf-int, max: inf-int]
```

Comment:
This looks like a policy construct which i am little nervous in absorbing here..?

is there a way to express it more clearly using base types?

Reply:
It's a weak form of comparison for use in conditional endorsements. We have SVN range comparisons. This is just slightly broader.